components/MaintainerCard.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <a
+    href="/grid"
+    class="maintainer"
+    :class="{ 'maintainer--oversize': oversize }"
+  >
+    <div class="maintainer__image">
+      <img
+        :src="
+          require(`~/assets/img/maintainers/${maintainer.src
+            .toLowerCase()
+            .replace(/\s+/g, '-')
+            .replace(/\./g, '')}.jpg`)
```

Comment:
💬 As mentioned in the other PR we could move this to the "data" file 👍 . This can be done in a later PR as a refactor.